tests/layer_tests/pytorch_tests/test_lstm_cell.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import pytest
+import torch
+import torch.nn as nn
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class aten_lstm_cell(torch.nn.Module):
+    def __init__(self, input_size, hidden_size, has_bias):
+        super().__init__()
+        self.cell = nn.LSTMCell(input_size, hidden_size, bias=has_bias)
+
+    def forward(self, x, h0, c0):
+        return self.cell(x, (h0, c0))
+
+
+class TestLSTMCell(PytorchLayerTest):
+
+    def _prepare_input(self, batch=3, input_size=10, hidden_size=20):
+        np.random.seed(42)
+        x = np.random.randn(batch, input_size).astype(np.float32)
+        h0 = np.random.randn(batch, hidden_size).astype(np.float32)
+        c0 = np.random.randn(batch, hidden_size).astype(np.float32)
+        #  Wrap hidden and cell states into a tuple
+        return (x, (h0, c0))
+
+    # ----------------------------------------------------------------------
+    # 1. Multi-step forward (smoke test)
+    # ----------------------------------------------------------------------
+    @pytest.mark.parametrize("has_bias", [True, False])
+    def test_lstm_cell_multistep_forward(self, ie_device, precision, ir_version, has_bias):
```

Comment:
```suggestion
    @pytest.mark.precommit
    @pytest.mark.nightly
    def test_lstm_cell_multistep_forward(self, ie_device, precision, ir_version, has_bias):
```
Otherwise tests will not run